src/pages/postgraphile/required-knowledge.md

```diff
@@ -176,3 +176,45 @@ create policy friend_insert on my_other_table for insert with check ( are_friend
   that action (doesn't make sense for `SELECT` or `DELETE`)
 - NOTE: if `with check` is not specified then the `using` clause will also be
   used for `with check` automatically.
+
+#### Recommended path
+
+A good way to tackle and make a sense of postgraphile is to go through these
+steps:
+
+**Introduction**
+
+https://www.graphile.org/postgraphile/introduction/
+
+Describes why you would want to go with this approach in the first place and
+reap the benefits and features that one solves using postgraphile. If you find
+this all daunting and too much `SQL`, do not worry. We do not usually use the
+database as a application, which it certainly should to reep all the benefits
+from being close to the data as well as the execution of the operations; more on
+the benefits is describe
+
+**Quick start**
+
+https://www.graphile.org/postgraphile/quick-start-guide/
+
+Walks you through installing the proper software like `node` and `postgres` and
+creating of a database that postgraphile will be hooking into.
+
```

Comment:
```suggestion
Walks you through installing the required software such as `node` and `postgres`
and creating of a database that PostGraphile will be connecting to.

```

Reply:
Multiline suggestion I love it!!!!!!!!!!!!!!